packages/app-utils/src/plugins/PluginUtils.tsx

```diff
@@ -86,7 +90,9 @@ export async function loadModulePlugins(
       if (module.status === 'fulfilled') {
         pluginMap.set(
           name,
-          isLegacyPlugin(module.value) ? module.value : module.value.default
+          isLegacyPlugin(module.value)
+            ? module.value
+            : module.value.default ?? module.value
```

Comment:
Not sure if I'm doing something wrong in my actual Plugin, but `default` is undefined despite me exporting as default.